### DISCUSSION TITLE
Add auto-firewall feature

### DISCUSSION THREAD

**Discussion by JoanFo1456** [pelican-dev/panel#630] [Ideas]
When you open a port in Pelican doesn't open it automatically and it can be a headache, so why not making it automatically when you create an allocate, manually or by auto allocate.
I think this can be a good idea, so it can be handled manually instead of going to open every port.

**Comment by andyondrya** [comment]
I feel like this would be weird to input because if you have the panel running as docker, you wouldn't be able to open externally.

**Comment by Boy132** [comment]
This is nothing Pelican can handle.
How and where to open ports depends on the specific system and its firewall(s).

**Comment by JoanFo1456** [comment]
But like a "config" for every system, like a setting on the admin panel, you input your system and then when an allocation is made, checks the system and uses the required command, if you open 25512 and you add on the admin panel that you're using ubuntu, then it uses ufw allow 25512.
I know this can be dumb, but can help some people i think.

**Reply by Boy132** [reply]
It's not that simple. And just displaying an ufw command won't help at all.
